test/functionalTest/cases/3707_get_or_change_log_config/get_or_change_log_config.test

```diff
@@ -0,0 +1,758 @@
+echo
+echo
+
+
+--REGEXPECT--
+01. GET initial log config
+==========================
+HTTP/1.1 200 OK
+Content-Length: 91
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "infoPayloadMaxSize": "5120",
+    "level": "DEBUG",
+    "lineMaxSize": "32768"
+}
+
+
+02. PUT /admin/log and see error
+================================
+HTTP/1.1 400 Bad Request
+Content-Length: 30
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "error": "log config missing"
+}
+
+
+03. GET log config, see initial log config
+==========================================
+HTTP/1.1 200 OK
+Content-Length: 91
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "infoPayloadMaxSize": "5120",
+    "level": "DEBUG",
+    "lineMaxSize": "32768"
+}
+
+
+04. PUT /admin/log?level=ERROR and see it work
+==============================================
+HTTP/1.1 200 OK
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+05. GET log config, see level=ERROR
+===================================
+HTTP/1.1 200 OK
+Content-Length: 91
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "infoPayloadMaxSize": "5120",
+    "level": "ERROR",
+    "lineMaxSize": "32768"
+}
+
+
+06. PUT /admin/log?level=xyz and see error
+==========================================
+HTTP/1.1 400 Bad Request
+Content-Length: 29
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "error": "invalid log level"
+}
+
+
+07. GET log config, see previous data
+=====================================
+HTTP/1.1 200 OK
+Content-Length: 91
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "infoPayloadMaxSize": "5120",
+    "level": "ERROR",
+    "lineMaxSize": "32768"
+}
+
+
+08. PUT /admin/log?level=INFO and see it work
+=============================================
+HTTP/1.1 200 OK
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+09. GET log config, see level=INFO
+==================================
+HTTP/1.1 200 OK
+Content-Length: 90
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "infoPayloadMaxSize": "5120",
+    "level": "INFO",
+    "lineMaxSize": "32768"
+}
+
+
+10. PUT /admin/log?infoPayloadMaxSize=5000 and see it work
+==========================================================
+HTTP/1.1 200 OK
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+11. GET log config, see infoPayloadMaxSize=5000
+===============================================
+HTTP/1.1 200 OK
+Content-Length: 90
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "infoPayloadMaxSize": "5000",
+    "level": "INFO",
+    "lineMaxSize": "32768"
```

Comment:
I thinks it would be better:

```suggestion
    "infoPayloadMaxSize": 5000,
    "level": "INFO",
    "lineMaxSize": 32768
```

I mean, lineMaxSize and infoPayloadMaxSize as JSON numbers instead of JSON string meaning numbers

(Sorry for not detecting this in previous review rounds...)

Reply:
fixed in [b72f2f3](https://github.com/telefonicaid/fiware-orion/pull/4289/commits/b72f2f32bebc08ed52276a61e8a43a851a78faed)